config/config.go

```diff
@@ -1435,3 +1443,26 @@ func (c *RemoteReadConfig) UnmarshalYAML(unmarshal func(interface{}) error) erro
 	}
 	return nil
 }
+
+// AzureServiceFabricSDConfig is the configuration for Azure Service Fabric service discovery.
+type AzureServiceFabricSDConfig struct {
```

Comment:
This doesn't look like the existing azure SD, why is it doing auth etc. differently?